src/types.ts

```diff
@@ -1,5 +1,13 @@
 import { SvgIconProps } from '@material-ui/core/SvgIcon';
 
+export interface TEnvVariables {
```

Comment:
can you add the others

Reply:
[shore thang](https://github.com/CodeForBoulder/upswyng/pull/78/commits/cfb821fb6807107cdc15eb6f2974412d9e5fa740)